### ISSUE TITLE
Bones stretching endlessly in play mode

### ISSUE THREAD

**Issue by AtriusX** [VRCFury/VRCFury#165]
Recently tried to update to the latest version from 1.506.0 and noticed that upon entering play mode one of my avatar's bones was stretching off into the distance endlessly. A similar issue occurred with this a while back on issue #60.

[Current picture for reference](https://github.com/VRCFury/VRCFury/assets/7331441/e8434b80-0a40-4c52-ba2e-79132dad553b)

**Comment by SenkyDragon** [comment]
This is most often caused by using WD off in an avatar with an additive layer. The simplest solution is to set your vrcfury Fix Write Defaults component to Force On

**Comment by AtriusX** [comment]
I can confirm that forcing WD on does indeed resolve the issue, but in this case I don't think this solution is going to work for me as parts of my avatar require WD off for state logic. I know this was working fine back on 1.506 at least, regardless of the WD setting. It seems like the presence of an additive layer doesn't really have any bearing on things either, as the problem still occurs when no additive layer is present.

**Comment by SenkyDragon** [comment]
Are you able to provide a project that reproduces the issue? https://vrcfury.com/discord

**Comment by AtriusX** [comment]
I did a bit more testing and I believe I was able to narrow the issue down to the problem being caused by a blend tree in the gesture layer that I wasn't actively using. I'm able to delete it so the problem is functionally solved for me at least.

I did a bit more digging and it appears that the issue might be related to the fact that VRCFury is moving all of the gesture controller layers into the FX controller. The version I was using previously kept the layers in the gesture controller. If you still would like a project though, I'll try to trim down mine a bit later when I have time.
